nix/default.nix

```diff
@@ -138,7 +139,7 @@ in rec {
   # DB Companion
   kibana = pkgsNonFree.kibana7.overrideAttrs (old: rec {
     version = elk7Version;
-    name = "kibana-${version}";
+    name = "kibana-7.17.5";
```

Comment:
you need to change the sha256 below, e.g. change a number to get the expected hash

Reply:
I don't change the version of kibana into that patch (it raises issue regarding nodejs ... it was the rabbit hole). So the hash is still the same.